northstar/src/runtime/process/init.rs

```diff
@@ -30,12 +30,36 @@ pub(super) struct Init {
     pub seccomp: Option<AllowList>,
 }
 
+/// Init process main function
+///
+/// This functions runs inside a multi-threaded tokio runtime.
+///
+async fn init_main(mut channel: Channel) {
+    loop {
+        match waitpid(None, None) {
```

Comment:
If the `exec` feature comes true the runtime should not consider the container as exited when the `exec` process exits. Probably extending the messages that is sent via the channel make sense e.g

```rust
enum Exit {
  // The command specified in the manifest as init. Oh man -  this is confusing
  Init(ExitStatus),
  Other(Pid, ExitStatus),
}

Reply:
yep, this adds a new dimension to consider in the start/stop/mount/unmout/shutdown cases :cry: